float32 time_diff
int32 wheel1_val
int32 wheel2_val
int32 wheel3_val
int32 wheel4_val